lisa/env.py

```diff
@@ -210,6 +212,46 @@ def from_one_conf(cls, path):
             plat_info = None
         return cls(target_conf=target_conf, plat_info=plat_info)
 
+    @classmethod
+    def from_cli(cls):
```

Comment:
Being able to specify the argument list can be handy:
```
def from_cli(argv=None):
```
`argv` can be given to `parse_args`, which will by itself default to sys.argv[1:] if `None` is given.

Since a --platform-info option is desirable (see below), that method does not fully belong to TargetConf so it's better for it to stay here in TestEnv.